architecture/topics/packet_flow.adoc

```diff
@@ -22,7 +22,7 @@ on a remote host on the cluster network. Then the flow of packets from container
 A to container B is as follows:
 
 *_eth0_* _(in A's netns) -> *vethA* -> *br0* -> *vxlan0* ->
-network footnote:[After this point, device names refer to devices on container
+network footnoteref:[After this point, device names refer to devices on container
 B's host.] -> *vxlan0* -> *br0* -> *vethB* -> *eth0* (in B's netns)_
```

Comment:
@jboxman i think the break to the newline might be confusing the build.

Reply:
@vikram-redhat, this looks like it's actually valid as originally written:

https://www.methods.co.nz/asciidoc/chunked/ch18.html

> footnote:[<text>]
Generates a footnote with text <text>.

Reply:
And this: https://docs.openshift.com/container-platform/3.11/architecture/networking/sdn.html#sdn-packet-flow